sdk/core/src/main/java/com/klaviyo/core/DeviceProperties.kt

```diff
@@ -69,11 +70,26 @@ object DeviceProperties {
         ).toString()
     }
 
-    val userAgent: String by lazy {
-        val sdkAgent = "klaviyo-${sdkName.replace("_", "-")}"
-        "$applicationLabel/$appVersion ($applicationId; build:$appVersionCode; $platform $osVersion) $sdkAgent/$sdkVersion"
+    val pluginSdk: String? by lazy {
+        Registry.config.applicationContext.getString(R.string.klaviyo_sdk_plugin_name_override)
+            .ifBlank { null }
     }
 
+    val pluginSdkVersion: String? by lazy {
+        Registry.config.applicationContext.getString(R.string.klaviyo_sdk_plugin_version_override)
+            .ifBlank { null }
+    }
+
+    val userAgent: String
+        get() {
+            val pluginString = pluginSdk?.let {
+                "($it/$pluginSdkVersion)"
```

Comment:
extremely nitpicky, but can we guard against this being null/empty better? even if you just put "unknown" or something, I think i'll work better for our regex/detection if we have `klaviyo-expo/SOMETHING` and never just `plugin/`